kubernetes/helm-charts/buildfarm/values.yaml

```diff
@@ -140,10 +140,10 @@ shardWorker:
     #   secretName: my-secret
     #   defaultMode: 0600
 
-  extraVolumeMounts: []
-    # - name: customConfig
-    #   mountPath: /mnt/config
-    #   readOnly: true
+  #extraVolumeMounts:
```

Comment:
Isn't this going to freak out with a now missing field? Not familiar enough with helm templating to know myself

Reply:
nah, the "with" form's block is omitted if referenced value is undefined.
I learned this trick from this helm chart [here](https://github.com/bazelbuild/bazel-buildfarm/blob/main/kubernetes/helm-charts/buildfarm/templates/server/deployment.yaml#L18). `podAnnotations` doesn't have a default [value](https://github.com/bazelbuild/bazel-buildfarm/blob/main/kubernetes/helm-charts/buildfarm/values.yaml) and still the template renders.